pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.3.3</version>
+    <version>2.3.5</version>
```

Comment:
This isn't strictly necessary but I generally like keeping up with upstream versions.